elements/dash-tree/demo.html

```diff
@@ -0,0 +1,103 @@
+<style>
+  dash-tree {
+    font-family: 'Courier New', Courier, monospace;
+  }
+  dash-tree,
+  dash-tree-item,
+  dash-tree-group,
+  label {
+    display: block;
+  }
+  dash-tree-group {
+    padding-left: 20px;
+  }
+  dash-tree-item[aria-expanded="false"] > label::before {
+    content: '+';
+  }
+  dash-tree-item[aria-expanded="true"] > label::before {
+    content: '-';
+  }
+  dash-tree-item[aria-expanded="true"] > dash-tree-group {
+    display: block;
+  }
+  dash-tree-item[aria-expanded="false"] > dash-tree-group {
```

Comment:
It may be worthwhile to add an expanded property to the treeitems, that would trigger the aria-expanded state when the element upgrades. This would be inline with the theory of keeping aria as an implementation detail that developers don't have to worry about